harness/attributes/common.yml

```diff
@@ -48,6 +48,9 @@ attributes.default:
     external_image_registries: []
     repository: = @("workspace.name")
     tagPrefix: = ''
+    platforms:
+      production:
+      - "linux/amd64"
```

Comment:
need to document these in https://github.com/inviqa/harness-go/blob/master/docs/harness-attributes.md